### PR TITLE
Update Specviz GUI data loading docs

### PR DESCRIPTION
**Pull request by rosteen**
Mostly updating screenshots to reflect the update UI, simple enough.

### CODE REVIEW

File: docs/specviz/import_data.rst [hunk 1]
```diff
@@ -26,11 +26,7 @@ and choose a file that can be parsed as a :class:`~specutils.Spectrum1D` in the
 
 .. image:: img/import_data_2.png
 
-After clicking :guilabel:`Import`, the data file will be parsed and loaded into the application. A notification will appear to let users know if the data import was successful. Afterward, the new data set can be found in the :guilabel:`Data` tab of each viewer's menu.
-
-.. image:: img/import_data_3.png
-
-Here, users can select the loaded data set to be visualized in the viewer.
+After clicking :guilabel:`Import`, the data file will be parsed and loaded into the application. A notification will appear to let users know if the data import was successful, and if possible the data will be immediately displayed (this may not be possible if an incompatible spectrum was already displayed). Afterward, the new data set can be found in the :guilabel:`Data` tab of each viewer's menu. Here, users can select which spectra that have be loaded to be visualized in the viewer.
```
Comment (pllim) [inline docs/specviz/import_data.rst:29]: Nitpick: Is it possible to break this paragraph into shorter lines so it is easier to review? Thanks!

File: docs/specviz/import_data.rst [hunk 2]
```diff
@@ -26,11 +26,12 @@ and choose a file that can be parsed as a :class:`~specutils.Spectrum1D` in the
...
 
-After clicking :guilabel:`Import`, the data file will be parsed and loaded into the application. A notification will appear to let users know if the data import was successful. Afterward, the new data set can be found in the :guilabel:`Data` tab of each viewer's menu.
-
-.. image:: img/import_data_3.png
-
-Here, users can select the loaded data set to be visualized in the viewer.
+After clicking :guilabel:`Import`, the data file will be parsed and loaded into the
+application. A notification will appear to let users know if the data import was 
+successful, and if possible the data will be immediately displayed (this may not 
+be possible if an incompatible spectrum was already displayed). Afterward, the new
```
Comment (pllim) [inline docs/specviz/import_data.rst:32]: Maybe clarify what one should expect when incompatible spectrum is displayed? Will it crash at loading/selection or something?

**Comment by pllim** [comment]
p.s. I wonder if we want to set `data_label` in the examples and screenshots so the Data dropdown not polluted by randomly generated labels.

p.p.s. I never knew you have to use `load_spectrum`. I always thought it is `load_data`.

**Review by pllim — APPROVED** [review]
Just nitpicks, otherwise LGTM. Thanks!

**Comment by rosteen** [comment]
@pllim On the one hand, I agree that the screenshot would look better with a good data label, but on the other hand you can't specify a label when loading from the GUI, so maybe the screenshot should be representative of what the user will see in that case?

**Comment by pllim** [comment]
> should be representative

Ah, the pesky standalone app. OK, that makes sense.

I already approved, so unless you want a second approval, feel free to merge when CI passes (ignore the dev job failure).

**Pull request merged**